cmd/openshift/operator/kodata/tekton-addon/addons/02-clustertasks/source_local/s2i-dotnet/s2i-dotnet-task.yaml

```diff
@@ -52,10 +56,27 @@ spec:
     - name: generate
       image: registry.redhat.io/ocp-tools-4-tech-preview/source-to-image-rhel8@sha256:98d8cb3a255641ca6a1bce854e5e2460c20de9fb9b28e3cc67eb459f122873dd
       workingDir: $(workspaces.source.path)
-      command: ['s2i', 'build', '$(params.PATH_CONTEXT)', 'image-registry.openshift-image-registry.svc:5000/openshift/dotnet:$(params.VERSION)', '--as-dockerfile', '/gen-source/Dockerfile.gen']
+      args: ["$(params.ENV_VARS[*])"]
+      script: |
+        echo "Processing Build Environment Variables"
+        echo "" > /env-vars/env-file
+        for var in "$@"
+        do
+            echo "$var" >> /env-vars/env-file
+        done
+
+        echo "Generated Build Env Var file"
+        echo "------------------------------"
+        cat /env-vars/env-file
```

Comment:
not sure if printing the environment variables is a good practice since it might contain sensitive information, wdyt? 🤔

Reply:
These are set as params on the task itself, so not sure if not printing them here would help. I thought that it might be better to print them for tackling problems/debugging.

If prefer to remove them, I can do it tomorrow earliest.

Reply:
@zroubalik yep, let's err on the side of caution here and remove the printing

Reply:
let's remove them in a follow up PR